### DISCUSSION TITLE
Team remarks / discussion in an exception

### DISCUSSION THREAD

**Discussion by fabricecw** [spatie/flareapp.io-roadmap#33] [Ideas]
Sometimes, there are exceptions which can't be resolved immediately or should remain open for a reason (example: we throw an exception when an external API couldn't be reached at the first time but it worked the second. The exception history gives us an indication of last API outages).

In this cases, a remarks or "discussion" section would help a lot. For example, you can give quick information about the exception context or that no immediate action is required.

**Comment by freekmurze** [comment]
Yeah, adding the ability to have a discussion on an exception makes sense, we'll look into that soon!

**Comment by fabricecw** [comment]
Thanks Freek! If I can assist you in any way, let me know.

**Comment by fabricecw** [comment]
Hi!

Just wanted to ask if this idea is already on the roadmap? Otherwise we have to enable team conversation about Flare apps in another way. Thanks for an update!

**Comment by AlexVanderbist** [comment]
We're currently working on this! ETA is a couple of weeks :)
